build-system/tasks/compile.js

```diff
@@ -45,9 +42,6 @@ exports.compile = async (options = {}) => {
   mkdirSync('build/fake-module/src');
   mkdirSync('build/css');
 
-  // Compile CSS because we need the css files in compileJs step.
-  await compileCss('./src/', './build/css', options);
-
   // Optionally skip JavaScript compilation.
   if (options.skipJs) {
```

Comment:
We could remove this option too. Unit tests would no longer need to call this method